components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -86,6 +82,8 @@ object Glean {
 
         initializeCoreMetrics(applicationContext)
 
+        Baseline.os.set("Android")
```

Comment:
Just an example to test that we can use a generated metric.  This isn't really where we would do this.

Reply:
Ah, ok. Let's either remove it or leave a comment stating that.

Reply:
I've moved this to `initializeCoreMetrics` and commented out the other ones.